models/repo_list.go

```diff
@@ -293,6 +293,17 @@ func userCollaborationRepoCond(idStr string, userID int64) builder.Cond {
 	)
 }
 
+// userCollaborationRepoCond returns a condition for selecting all repositories a user is collaborator in
+func userCollaborationRepoCond(idStr string, userID int64) builder.Cond {
+	return builder.In(idStr, builder.Select("repo_id").
+		From("`collaboration`").
+		Where(builder.And(
+			builder.Eq{"`collaboration`.user_id": userID},
+			builder.Gt{"`collaboration`.mode": int(perm.AccessModeNone)},
```

Comment:
I don't think you can create a collaboration with AccessModeNone so I'm not sure that this is needed.

Reply:
You are probably right. In the user interface the options are `Admin`, `Write` and `Read` access. I will remove it.

Reply:
Done